### PR TITLE
Mail 47 donation funds thanks

### PR DESCRIPTION
**Pull request by bdsl**

### CODE REVIEW

File: templates/_base.html.twig [hunk 1]
```diff
@@ -53,7 +53,11 @@
         <p>
           Kind regards,<br>
           Big Give Team<br>
-          BigGive.org<br>
+          <br>
+          Big Give, Dragon Court, 27-29 Macklin Street, London, WC2B 5LX<br>
+          Email: hello@biggive.org<br>
+          Web: http://biggive.org<br><br>
```
Comment (NoelLH) [inline templates/_base.html.twig:59]:
```suggestion
          Web: https://biggive.org<br><br>
```

File: templates/_base.text.twig [hunk 2]
```diff
@@ -2,7 +2,10 @@
 {% block footer %}
     Kind regards,
     Big Give Team
-    BigGive.org
+
+    Big Give, Dragon Court, 27-29 Macklin Street, London, WC2B 5LX
+    Email: hello@biggive.org   Web: http://biggive.org
```
Comment (NoelLH) [inline templates/_base.text.twig:7]:
```suggestion
    Email: hello@biggive.org   Web: https://biggive.org
```

File: tests/templates/__snapshots__/DonorFundsThanksTest__testItRendersEmailToThankDonorForFundingAnAccount__1.txt [hunk 3]
```diff
@@ -27,4 +27,7 @@
     </p>
     Kind regards,
     Big Give Team
-    BigGive.org
+
+    Big Give, Dragon Court, 27-29 Macklin Street, London, WC2B 5LX
+    Email: hello@biggive.org   Web: http://biggive.org
```
Comment (NoelLH) [inline tests/templates/__snapshots__/DonorFundsThanksTest__testItRendersEmailToThankDonorForFundingAnAccount__1.txt:32]:
```suggestion
    Email: hello@biggive.org   Web: https://biggive.org
```

**Comment by bdsl** [comment]
Thanks @NoelLH - I applied your suggestions.

**Review by NoelLH — APPROVED** [review]

**Pull request merged**